specification/network/resource-manager/Microsoft.Network/stable/2020-11-01/virtualWan.json

```diff
@@ -6105,6 +6105,10 @@
             "$ref": "#/definitions/VpnGatewayIpConfiguration"
           }
         },
+        "enableBgpRouteTranslationForNat": {
```

Comment:
>enableBgpRouteTranslationForNat [](http://example.com/codeflow?start=9&length=31)

How does your RP handle downlevel clients?

ARM recommendation is to change the API version when adding new properties to an existing API. It's not a hard requirement, but it can break down-level clients, depending on the service behavior. For example:
1. Portal or new client PUTs the resource, including the new properties
2. Downlevel client PUTs/PATCHs the same resource without the new properties.

What state will the resource be in afterward? Will it make sense to customers from the perspective of both clients and the portal?
Without incrementing the version, your service can't control that situation, e.g. rejecting down-level PUT/PATCH.